docs/community/old-setup/old-hostsystem.md

```diff
@@ -1,169 +0,0 @@
-
-Due to frequent access, using `atime` updates can significantly reduce the performance
-of FeatureBase. `noatime` should be used on the mount point for the FeatureBase data directory.
-
-```
-  #!/bin/bash
-
-  DATA_DIR="/opt/molecula/featurebase"
-
-  PARTITION=$(df $DATA_DIR | awk 'NR==2 { print $1; exit }')
-
-  MOUNT_POINT=$(df $DATA_DIR | awk 'NR==2 { print $6; exit }')
-
-  MOUNT_OPTIONS=$(findmnt --source $PARTITION --output=options | sed -n '2p')
-
-  if [[ "$MOUNT_OPTIONS" == *noatime* ]]; then
-      echo "noatime is already enabled!"
-  elif [[ $(lsblk $PARTITION -n --output=FSTYPE) == "xfs" && "$MOUNT_OPTIONS" == *relatime* ]]; then
-      echo "reltime is enabled, mount with noatime to improve performance; to mount with noatime:"
-      echo "    mount -o remount,noatime $PARTITION $MOUNT_POINT"
-  else
-      echo "noatime should be enabled on the mount point for the data directory; to mount with noatime:"
-      echo "    mount -o remount,noatime $PARTITION $MOUNT_POINT"
-  fi
-
-```
-
-#### Enable TRIM on SSDs
-
-On SSDs, TRIM should be enabled to reclaim the memory blocks that are no longer considered to be 'in use'.
-
-```
-  #!/bin/bash
-
-  DATA_DIR="/opt/molecula/featurebase"
-
-  PARTITION=$(df $DATA_DIR | awk '{print $1}' | sed -n '2p')
-
-  MOUNT_OPTIONS=$(findmnt --source $PARTITION --output=options | sed -n '2p')
-
-  if [[ $(lsblk $PARTITION -n -o ROTA) == "1" ]]; then
-      echo "The partition is on a rotational drive, TRIM not supported"
-  elif [[ $(lsblk $PARTITION -n -o DISC-GRAN == "0B") ]]; then
-      echo "DISC granularity 0, TRIM not supported"
-  elif [[ $(systemctl is-enabled fstrim.timer) == "enabled" ]]; then
-      echo "fstrim is already enabled!"
-  else
-      echo "fstrim must be enabled; to enable:"
-      echo "    sudo systemctl enable fstrim.timer"
-  fi
-
-  if [[ "$MOUNT_OPTIONS" == *discard* ]]; then
-      echo "Continuous TRIM(discard) is not recommended, switch to periodic TRIM(fstrim)"
-  fi
- ```
-
-### Mac OS Systems
-
-On Mac OS, `ulimit` does not behave predictably, and the way to change the limits changes between releases. Additionally, the system's system integrity protection (SIP) functionality may prevent attempts to change this. For a more detailed writeup, have a look at Wilson Mar's [maximum limits](https://wilsonmar.github.io/maximum-limits/) page. The number of processes limit described there shouldn't be significant for FeatureBase. Mac OS does not seem to have a direct parallel to the maximum map count limits found on Linux systems.
-
-You can temporarily raise the open file limit using `launchctl limit maxfiles 262144 262144`, although this will not persist across reboots. To persist it across reboots, you need to cause this command to be run during system startup every time. The easiest way to do this is by adding things to `/Library/LaunchDaemons`, but you can't do this while SIP is enabled, and you can't run `csrutil` once the system is up and running.
```

Comment:
why was this content no included in the new stuff?

Reply:
Because it's documented in greater detail in externally managed help. Plus the instructions are different based on which MacOS you use.
That's why I've added links out rather than maintaining our own copy of docs which we'd need to keep up to date.

Reply:
I don't think the external help references the actual number we are looking for that's here, does it? If not we should definitely keep it